src/main/java/de/diddiz/LogBlock/blockstate/BlockStateCodecSign.java

```diff
@@ -92,7 +91,7 @@ public void deserialize(BlockState state, YamlConfiguration conf) {
         if (state instanceof Sign) {
             Sign sign = (Sign) state;
             if (conf != null) {
-                sign.setEditable(!conf.getBoolean("waxed"));
+                sign.setWaxed(!conf.getBoolean("waxed"));
```

Comment:
I think this should be `sign.setWaxed(conf.getBoolean("waxed"));`. setEditable is the opposite of setWaxed